lib/components/Modal/Modal.tsx

```diff
@@ -33,7 +33,7 @@ export const Modal = ({ open, onClose, content, progress, sidePanel, footer }: M
       {/* Wrapper container paddings and margins */}
       <div className="ds-fixed ds-inset-0">
         {/* Wrapper for container centering */}
-        <div className="ds-flex ds-items-center ds-justify-center ds-h-full ds-m-3">
+        <div className="ds-flex ds-items-center ds-justify-center ds-h-full">
```

Comment:
Margin removed because it caused the actual content to not be centered on the screen (gaps in top and bottom of the screen were not 40px each if this margin was in place).